"""Exceptions that can be thrown by identity."""


class VerifyError(Exception):
    """An Exception occurred when a verification process failed."""

    pass
